src/clusterfun_server_main.ts

```diff
@@ -81,7 +83,7 @@ app_ws.app.ws('/talk/:roomId/:personalId', api.handleSocket);
 // The deploy system should instead either deploy the frontend and games to this
 // "client" folder in dist, or the server should determine the path to the lobby from
 // a manifest file
-const clientPath = "client"
+const clientPath = isDev ? "../../clusterfun-client/build" : "client"
```

Comment:
Could we go ahead and make clientPath an environment variable? I would really like to avoid bringing this back.